libraries/AdaptiveExpressions/BuiltinFunctions/Ignore.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AdaptiveExpressions.Memory;
+
+namespace AdaptiveExpressions.BuiltinFunctions
+{
+    public class Ignore : ExpressionEvaluator
```

Comment:
>Ignore [](start = 17, length = 6)

Documentation.  Mark a clause so that MostSpecificSelector will ignore it.  MostSpecificSelector considers A & B to be more specific than A, but some clauses are unique and incomparable.